packages/amplify_api/lib/model_queries.dart

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_api/amplify_api.dart';
+import 'package:amplify_api/graphql_helpers.dart';
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+
+// This class provides static method calls to enable a simpler DX
+class ModelQueries {
+  static GraphQLRequest<T> get<T extends Model>(
+      ModelType<T> modelType, String id) {
+    return ModelQueriesFactory().get<T>(modelType, id);
```

Comment:
This is totally fine for now- we can work on how to improve this in the next go-around, since we don't want to be creating a new instance in every method or every time this method is called.

Reply:
Great, I'll be addressing this in my next PR!